commands/common.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) OpenFaaS Project 2017. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"errors"
+)
+
+func validateLanguageFlag(language string) (string, error) {
+	var err error = nil
+
+	if language == "Dockerfile" {
+		language = "dockerfile"
+
+		err = errors.New("Dockerfile was converted to dockerfile")
```

Comment:
What is this error for? ( I thought we were re-moving the message? )

Reply:
Just in case we need to display the event, it's not a real error, just a notice to know that we modified the language variable